test/parallel/test-bootstrap-modules.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable node-core/required-modules */
+
+'use strict';
+
+// Ordinarily test files must require('common') but that action causes
+// the global console to be compiled, defeating the purpose of this test.
+// This makes sure no additional files are added without carefully considering
+// lazy loading. Please adjust the value if necessary.
+
+const assert = require('assert');
+
+assert(process.moduleLoadList.length <= 73, process.moduleLoadList);
```

Comment:
A better way to test this would be to start a child process printing `process._rawDebug(JSON.stringify(process.moduleLoadList))` to stdout to reduce the noise.

Reply:
In what way does the current way produce noise?

Reply:
You won't need to `require('assert')`, which pollutes the `moduleLoadList`, I guess.

Reply:
Hm, calling `assert` could indeed pollute the list. Even though that is currently not the case. But that can be easily worked around by writing:

```
const list = process.moduleLoadList;

const assert = require('assert');

assert(list <= 73, list);
```

Reply:
@BridgeAR Using a child process also makes the `eslint-disable node-core/required-modules` and the comment explaining why unnecessary.